lib/ansible/modules/network/vyos/vyos_config.py

```diff
@@ -95,8 +96,23 @@
     choices: ['yes', 'no']
 """
 
+EXAMPLES = """
+tasks:
```

Comment:
Please remove `tasks:` and remove the leading two spaces from the rest of the examples.